sql/core/src/main/scala/org/apache/spark/sql/execution/metric/SQLMetrics.scala

```diff
@@ -233,4 +228,13 @@ object SQLMetrics {
         SparkListenerDriverAccumUpdates(executionId.toLong, metrics.map(m => m.id -> m.value)))
     }
   }
+
+  private def attachTaskId(name: String, conf: SparkConf): String = {
```

Comment:
We are attaching `stageId` and `attemptId`, too.

Reply:
Yeah, I know. I'm thinking about a better name, while adding stageId/attemptId makes me feel too long for the method name.

Reply:
`attachTaskInfo`? stage id is part of the task info which tells which stage the task comes from.